src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -78,9 +81,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
 
-    Message message = new Message(user, text);
-    datastore.storeMessage(message);
+    //Perform sentiment analysis when the user submits a message
```

Comment:
space after //. here and elsewhere
```suggestion
    // Perform sentiment analysis when the user submits a message
```